addons/website/static/src/snippets/s_dynamic_snippet_carousel/000.js

```diff
@@ -26,6 +26,7 @@ const DynamicSnippetCarousel = DynamicSnippet.extend({
                 interval: parseInt(this.el.dataset.carouselInterval),
                 rowPerSlide: parseInt(uiUtils.isSmall() ? 1 : this.el.dataset.rowPerSlide || 1),
                 arrowPosition: this.el.dataset.arrowPosition || '',
+                scrollMode: this.scrollMode = this.el.dataset.scrollMode || '',
```

Comment:
what is the goal of the `this.scrollMode =` here ? why not only `scrollMode: this.el.dataset.scrollMode || '',`

Reply:
mistake sorry